panel/tests/widgets/test_select.py

```diff
@@ -230,10 +230,11 @@ def test_nested_select_defaults(document, comm):
             "windspeed": [700, 500, 300],
         },
     }
-    select = NestedSelect(options=options)
-    assert select.value == ("Andrew", "temp", 1000)
+    levels = ["Person", "Variable", "Height"]
```

Comment:
I forget whether I made levels required. If not, maybe this shouldn't be set since it's `test_nested_select_defaults`
```suggestion
    levels = ["Person", "Variable", "Height"]
```